frontend/Pipfile

```diff
@@ -0,0 +1,19 @@
+[[source]]
+name = "pypi"
+url = "https://pypi.org/simple"
+verify_ssl = true
+
+[dev-packages]
+
+[packages]
+Jinja2 = "*"
+watchdog = "*"
+pyyaml = "*"
+argh = "*"
+flask = "*"
+requests = "*"
+
+[scripts]
+build = "python3 build.py ../config.json.example"
```

Comment:
I don't see `build.py` in this PR, did you forget to commit it?

Reply:
there were simply no changes to build.py

Reply:
🤦‍♂️  right

Reply:
note: we could theoretically simplify `build.py`, but I found it to be a bit complicated and I didn't feel comfortable to touch it because it still works like a charm